processes/ingest/loc.py

```diff
@@ -1,290 +1,114 @@
-                '{"catalog": false, "download": true, "reader": false, "embed": false}'
-            ])
-            record.has_part.append(linkString)
-
-        if 'epub_file' in resultsRecord['resources'][0].keys():
-            linkString2 = '|'.join([
-                '1',
-                resultsRecord['resources'][0]['epub_file'],
-                'loc',
-                'application/epub+zip',
-                '{"reader": false, "catalog": false, "download": true}'
-            ])
-            record.has_part.append(linkString2)
-
-
-    def storePDFManifest(self, record):
-        for link in record.has_part:
-            itemNo, uri, source, mediaType, flags = link.split('|')
-
-            if mediaType == 'application/pdf':
-                recordID = record.identifiers[0].split('|')[0]
-
-                manifestPath = 'manifests/{}/{}.json'.format(source, recordID)
-                manifestURI = 'https://{}.s3.amazonaws.com/{}'.format(
-                    self.s3Bucket, manifestPath
-                )
-
-                manifestJSON = self.generateManifest(record, uri, manifestURI)
-
-                self.s3_manager.createManifestInS3(manifestPath, manifestJSON, self.s3Bucket)
-
-                linkString = '|'.join([
-                    itemNo,
-                    manifestURI,
-                    source,
-                    'application/webpub+json',
-                    '{"catalog": false, "download": false, "reader": true, "embed": false}'
-                ])
-                record.has_part.insert(0, linkString)
-                break
-
-    def storeEpubsInS3(self, record):
-        for epubItem in record.has_part:
-            itemNo, uri, source, mediaType, flagStr = epubItem.split('|')
-
-            if mediaType == 'application/epub+zip':
-
-                recordID = record.identifiers[0].split('|')[0]
-
-                bucketLocation = 'epubs/{}/{}.epub'.format(source, recordID)
-                self.addEPUBManifest(
-                    record, itemNo, source, flagStr, mediaType, bucketLocation
-                )
-
-                self.rabbitmq_manager.sendMessageToQueue(self.fileQueue, self.fileRoute, get_file_message(uri, bucketLocation))
-                break
-
-    def addEPUBManifest(self, record, itemNo, source, flagStr, mediaType, location):
-            s3URL = 'https://{}.s3.amazonaws.com/{}'.format(self.s3Bucket, location)
-
-            linkString = '|'.join([itemNo, s3URL, source, mediaType, flagStr])
-
-            record.has_part.insert(0, linkString)
+            self.rabbitmq_manager.sendMessageToQueue(self.file_queue, self.file_route, get_file_message(epub_part.url, epub_location))
 
     @staticmethod
-    def generateManifest(record, sourceURI, manifestURI):
-        manifest = WebpubManifest(sourceURI, 'application/pdf')
+    def generate_manifest(record: Record, source_uri: str, manifest_uri: str):
+        manifest = WebpubManifest(source_uri, 'application/pdf')
```

Comment:
This manifest method as well has been moved to the s3Manager due to recent changes I merged into the main branch.

Reply:
Good shout! I'll get this up to date with the latest changes :)